src/backend/CassandraBackend.h

```diff
@@ -152,6 +153,25 @@ class CassandraStatement
         curBindingIndex_++;
     }
 
+    void
+    bindNextString(std::string const& data)
+    {
+        if (!statement_)
```

Comment:
Not your fault but this is basically duplicated in each function here.. we should extract this into a helper

Reply:
agreed and i think well out of scope for this pr